### DISCUSSION TITLE
再见2022 | 继续前行 - ek1ng's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ek1ng/blog#43] [Announcements]
# 再见2022 | 继续前行 - ek1ng's Blog

2022！ - ek1ng - ek1ng's Blog

https://www.ek1ng.com/%E5%86%8D%E8%A7%812022_%E7%BB%A7%E7%BB%AD%E5%89%8D%E8%A1%8C.html

**Comment by songhn233** [comment]
加油哦，虽然大环境很差，但是祝你好运🍀

**Reply by ek1ng** [reply]
谢谢学长！有空回学校的话可以一块吃个饭聊聊天呀

**Comment by SageKwun** [comment]
加油哦，会越来越好的！

**Comment by jalenzz** [comment]
e哥好强🤩
